redbeams/src/main/java/com/sequenceiq/redbeams/converter/stack/AllocateDatabaseServerV4RequestToDBStackConverter.java

```diff
@@ -78,8 +82,10 @@ public DBStack convert(AllocateDatabaseServerV4Request source, String ownerCrnSt
             dbStack.setParameters(parameter);
         }
 
+        long now = clock.getCurrentInstant().getEpochSecond();
```

Comment:
NIT: The fact that this is in seconds is surprising, so I would probably rename the variable for clarity.

Reply:
Heh, actually, this is sort of a bug. Here, the DBStackStatus created field gets a value in seconds, but DBStackStatusUpdater then uses milliseconds. I'll change this. The tag timestamp needs to use seconds for the time being, to match what happens for cluster instances. However, DBStackStatus should just use milliseconds.